packages/ui-extensions/src/surfaces/checkout/targets.ts

```diff
@@ -726,10 +733,10 @@ export interface RenderExtensionTargets {
     AnyComponent
   >;
   /**
-   * A static extension target that is rendered on top of the Checkout page as an overlay.
+   * A static extension target that is rendered on top of the **Thank you** page as an overlay.
    */
-  'purchase.checkout.overlay.render': RenderExtension<
-    CheckoutApi & StandardApi<'purchase.checkout.overlay.render'>,
+  'purchase.thank-you.chat.render': RenderExtension<
```

Comment:
The thank-you target was not added here, only in our internal version. This PR corrects that.